packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -896,10 +914,7 @@ class SortedTable extends Component {
                   paginationInstance
                 ))}
             </Col>
-            <Col mediumSize={4}>
-              {filterContainer ? <Portal container={() => filterContainer()}>{filterInstance}</Portal> : filterInstance}
-            </Col>
-            <Col mediumSize={1} className='pull-right'>
+            <Col mediumSize={5} className='pull-right'>
               {itemsPerPageContainer !== undefined ? (
                 <Portal container={() => itemsPerPageContainer()}>{dropdownItemsPerPage}</Portal>
```

Comment:
I think we should center the pagination and align the "items per page" dropdown with the right edge of the table now.